<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TechnicienWindow</class>
 <widget class="QDialog" name="TechnicienWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1292</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(125, 54, 195);
font: 900 16pt &quot;Arial Black&quot;;
</string>
   </property>
   <property name="text">
    <string>Transférer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>108</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 22pt &quot;Monotype Corsiva&quot;;
color: rgb(170, 85, 255)</string>
   </property>
   <property name="text">
    <string>Technicien</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>0</y>
     <width>357</width>
     <height>37</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 24pt &quot;Bell MT&quot;;
color: rgb(170, 85, 255)</string>
   </property>
   <property name="text">
    <string>Bonjour, technicien NOM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(125, 54, 195);
font: 900 16pt &quot;Arial Black&quot;;
</string>
   </property>
   <property name="text">
    <string>Clore</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(125, 54, 195);
font: 900 16pt &quot;Arial Black&quot;;
</string>
   </property>
   <property name="text">
    <string>Répondre</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>0</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>35</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>0</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>123</x>
     <y>0</y>
     <width>20</width>
     <height>51</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>80</y>
     <width>116</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 900 18pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Tickets</string>
   </property>
  </widget>
  <widget class="QPushButton" name="create_usr_2">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>30</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Se déconnecter</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>120</y>
     <width>501</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>251</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>490</y>
     <width>391</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>460</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Réponse :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
